### DISCUSSION TITLE
Run JupyterBook code cell with a local kernel

### DISCUSSION THREAD

**Discussion by davidcb98** [executablebooks/meta#1112] [Q&A]
Good morning,

I have been looking for information on how to change the kernel that Thebe uses to run the cells in interactive mode. By default it uses the MyBinder online service. Is there any way to tell JupyterBook to use another kernel? For example, a local one from my computer.

Best regards,
Thanks

**Comment by agoose77** [comment]
There is a demo page that shows the appropriate configuration for using a local notebook server:
https://github.com/executablebooks/thebe/blob/c085abe7752b8bd5a289352167105b8bebd51e61/packages/thebe/development/local.html#L20

However, this is for the newest version of thebe. My understanding is that the old version of thebe used by JB doesn't yet support local servers. @stevejpurves would you be able to chime in here perhaps?

If so, it would be possible to work around this, but it might be slightly painful to do so. So, let's see what @stevejpurves thinks!
